### PR TITLE
Update Nympho.yml

### PR DESCRIPTION
**Pull request by JaseNZC**
Was a fix from here.
#123 
Is a bit thrown together so feel free to adjust as you see fit.

### CODE REVIEW

**Comment by JaseNZC** [comment]
Changed incase they migrate all sites scenes patch.

**Comment by bnkai** [comment]
For the studio can now use
```yaml
      Studio:
        Name: //a[contains(@class,"navbar-brand")]/img/@title
```
No need to have the whole regex and weird (:-P) map.

**Comment by JaseNZC** [comment]
> For the studio we can now use
> 
> ```yaml
>       Studio:
>         Name: //a[contains(@class,"navbar-brand")]/img/@title
> ```
> 
> No need to have the whole regex and weird (:-P) map.

Updated , even better and more cleaner :-)

File: scrapers/Nympho.yml [hunk 1]
```diff
@@ -20,22 +20,15 @@ xPathScrapers:
...
         postProcess:
           - replace:
               - regex: (\d+)(?:[dhnrst]+)\s+(.+)
                 with: "$1 $2"
           - parseDate: 2 Jan 2006
-      Image: //video[contains(@id,"ypp-player")]/@poster|//div[@id="main-player"]/@data-screenshot
-# Last Updated December 06, 2021
+      Image: //video[contains(@id,"ypp-player")]/@poster|//div[@id="main-player"]/@data-screenshot|//meta[@property="og:image"]/@content
+driver:
+  useCDP: true
```
Comment (bnkai) [inline scrapers/Nympho.yml:33]: I didnt notice that before...
Why was cdp added here?

Comment (JaseNZC) [reply scrapers/Nympho.yml]: Have removed it, Think I had it in there as sometimes it would block you. I think it looks more like a cookie thing.

**Pull request merged**